src/metaschema/readme.md

```diff
@@ -4,23 +4,23 @@ An OSCAL Metaschema defines the the information structures that define an OSCAL
 
 OSCAL Metaschemas are used to generate other OSCAL artifacts based on the metaschema description. These artifacts include:
 
-- XML and JSON Schemas, or *data models*, are derived from the metaschema, expressed as XSD and JSON Schema;
+- XML and JSON schemas, or *data models*, which are derived from the metaschema and expressed as XSD and JSON Schema;
 - Conversion scripts that can convert OSCAL content between the XML and JSON OSCAL formats, ensuring the resulting content is schema valid;
 - XML and JSON [model documentation](https://pages.nist.gov/OSCAL/docs/schemas/) for use on the [OSCAL website](https://pages.nist.gov/OSCAL).
 
-Use of Metaschemas in OSCAL allow us to maintain seamless and consistent support for multiple OSCAL model formats, including XML and JSON. Content can also be kept up-to-date in multiple formats using generated content converters, and can be validated using generated schema. Adding support for new formats (e.g., YAML) can accomplished by extending the Metaschema tooling to produce schema and converters for other formats.
+Use of Metaschemas in OSCAL allows us to maintain seamless and consistent support for multiple OSCAL model formats, including XML and JSON. Content can also be kept up-to-date in multiple formats using generated content converters, and can be validated using generated schemas. Adding support for new formats (e.g., YAML) can accomplished by extending the Metaschema tooling to produce schemas and converters for other formats.
 
 The Metaschema syntax (an XML application) is also described and constrained with its [own schema](https://github.com/usnistgov/metaschema/blob/main/toolchains/xslt-M4/validate/metaschema.xsd), and with a [Schematron constraints set](https://github.com/usnistgov/metaschema/blob/main/toolchains/xslt-M4/validate/metaschema-composition-check.sch). The latter is able to enforce some of the rules described below.
```

Comment:
Blocking: can you remove this line **and everything inclusive below it**, I would prefer to remove all of it. (The GitHub ````suggestion` notation will not support the large suggestion to delete it all, I tried multiple times.)

```suggestion
```

Reply:
I removed all that content, in 11dfbb82a7f00973a0cfd911911a0a2ecc27214e.

Thanks for pointing me to another resource for my learning about Metaschema. :)